### ISSUE TITLE
Insufficient contrast in comments toolbar

### ISSUE THREAD

**Issue by pylapp** [Impressia/Impressia#165]
# Description

_As a user,
When I click on a comment,
The displayed toolbar is not that much readable_

# Details

The contrast ratio does not match the WCAG AA and AAA ratios.

## Light mode

<img height="500" src="https://github.com/user-attachments/assets/fb839a77-1bd7-4b08-9768-16bae6b524a3"/>

<img height="500" alt="Image" src="https://github.com/user-attachments/assets/1f392b63-9541-4b05-8799-19242e94b478" />

> [!NOTE]
> Ratio is 1.54:1, so not WCAG AA and AAA compliant.

## Dark mode

<img height="500" src="https://github.com/user-attachments/assets/d91efb56-bdfb-44e3-89a3-9be5ae575cf4"/>

<img height="500" alt="Image" src="https://github.com/user-attachments/assets/c47e0807-15d0-4860-b5a5-3e3e699592a4" />

> [!NOTE]
> Ratio is 2.6:1, so not WCAG AA and AAA compliant.

# Suggestions

> [!TIP]
> If we consider images in use are large text, it should be 3:1 (AA) or 4.5:1 (AAA)

# Versions

- iOS 26.4
- Impressia 3.2.0 (437) (from AppStore)
